controllers/users/authenticate-web.js

```diff
@@ -1,12 +1,9 @@
-const moment = require('moment');
-
 const UsersFunction = require('../../databases/functions/users');
-const {sequelize} = require('../../databases/models');
 
 const {User} = require('../../databases/models');
-const BetterSocialCore = require('../../services/bettersocial');
 const {createRefreshToken} = require('../../services/jwt');
 const Getstream = require('../../vendor/getstream');
+const HumanIdService = require('../../vendor/humanid');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import changes seem fine. The removal of unused imports `moment`, `sequelize`, and `BetterSocialCore` is a good practice for code cleanliness and performance. The addition of `HumanIdService` seems necessary for the new authentication process.

```diff
- const moment = require('moment');
- const {sequelize} = require('../../databases/models');
- const BetterSocialCore = require('../../services/bettersocial');
+ const HumanIdService = require('../../vendor/humanid');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->